client/src/components/select/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import '../../App.css';
+import './select.css';
+
+class Select extends Component {
```

Comment:
You might already be planning this, but when I'm making generic select / input / radio components, I usually have the component that's above it handle most of the logic by passing in the options to display, the current selection (or input value) in the state, and a click / change callback, all as props.
Then the Select or Input component is a very simple component whose job is basically just to look a certain way / be visually consistent, and the component that's including it, like the Login component for example, is handling all of the redux wiring (since the behaviors need to be so specific to different use cases anyways).